docs/development.rst

```diff
@@ -0,0 +1,102 @@
+Development
+============
+This chapter contains information helpful when you want to engage in development of this project.
+
+Prerequisites
+-------------
+If you want to engage in development of this project you should have the following libraries and tools available.
+
+Tools
++++++
+* python_ >= 3.6
+* git_
+* Docker_
+* ITDE_
+    * Prerequisites_
+
+Libraries
++++++++++
+* unixodbc
+* unixodbc-dev
+* libboost-date-time-dev
+* libboost-locale-dev
+* libboost-system-dev
+
+
+Example: Install of required system libraries on Ubuntu
+
+.. code-block::
+
+    sudo apt-get install unixodbc unixodbc-dev libboost-date-time-dev libboost-locale-dev libboost-system-dev
+
+
+Locale
++++++++
+Make sure the local is setup appropriately.
+
+Example: Setting up an english locale
+
+.. code-block::
+
+    sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && locale-gen
```

Comment:
This can be executed on a normal system without root. I guess you needed that in the ITDE

Reply:
Actually they used it in their docker containers. In the README.rst (notes) it is stated that it seems to prevent certain issues.